.cirrus.yml

```diff
@@ -297,14 +297,15 @@ task:
         - dart --enable-asserts dev/customer_testing/run_tests.dart --skip-on-fetch-failure --skip-template bin/cache/pkg/tests/registry/*.test
 
     # firebase_test_lab_tests are linux-only
-    - name: firebase_test_lab_tests-0-linux
-      <<: *FIREBASE_SHARD_TEMPLATE
+    $ TODO(tvolkert): Re-enable (https://github.com/flutter/flutter/issues/62427)
```

Comment:
This needs a `#` instead of a `$`

Reply:
thx.  Was in meetings so took a while to update.